bootkube-up

```diff
@@ -255,10 +236,10 @@ sleep 10
 ### WAIT FOR KUBERNETES API TO COME UP CLEANLY, THEN APPLY FOLLOWING LABELS AND MANIFESTS:
 echo_green "\nPhase X: Cluster created, and now deploying requested SDN along with additional labels:"
 sudo kubectl --kubeconfig=/etc/kubernetes/kubeconfig label node --all node-role.kubernetes.io/$KUBE_SDN-node=true --overwrite
-sudo kubectl --kubeconfig=/etc/kubernetes/kubeconfig label node --all node-role.kubernetes.io/master="" --overwrite
-sed "s?KUBE_POD_CIDR?${KUBE_POD_CIDR}?g; s?ETCD_SVC_ADDR?${ETCD_SVC_ADDR}?g; s?ETCD_SVC_PORT?${ETCD_SVC_PORT}?g; s?ETCD_SVC_PEER?${ETCD_SVC_PEER}?g; s?KUBE_SVC_PORT?${KUBE_SVC_PORT}?g" $BOOTKUBE_DIR/bootkube-ci/deploy-sdn/$KUBE_SDN/calico-v1.7.5.yaml  >  $BOOTKUBE_DIR/bootkube-ci/deploy-sdn/$KUBE_SDN/calico-bootkube-v1.7.5.yaml
+sed "s?KUBE_POD_CIDR?${KUBE_POD_CIDR}?g; s?ETCD_SVC_ADDR?${ETCD_SVC_ADDR}?g; s?ETCD_SVC_PORT?${ETCD_SVC_PORT}?g; s?ETCD_SVC_PEER?${ETCD_SVC_PEER}?g; s?KUBE_SVC_PORT?${KUBE_SVC_PORT}?g" $BOOTKUBE_DIR/bootkube-ci/deploy-sdn/$KUBE_SDN/calico-${KUBE_SDN_VERSION}-v1.7.5.yaml  >  $BOOTKUBE_DIR/bootkube-ci/deploy-sdn/$KUBE_SDN/calico-${KUBE_SDN_VERSION}-bootkube-v1.7.5.yaml
```

Comment:
i don't think all of these variables are bing used.

Example from the previous ConfigMap manifest:
```
data:
  # The location of your etcd cluster.  This uses the Service clusterIP
  # defined below.
  etcd_endpoints: "http://ETCD_SVC_ADDR:ETCD_SVC_PORT"
```

We should keep the same level of flexibility in the Calico manifest. Once we get Calico solid, I can go back and repair the Canal, since that will be really useful to explore for our team (differences between L2 and L3).

Reply:
good call